File: TemplateMapper/TemplateMapper/Models/Drawing.cs
using System;
using System.Collections.Generic;
using System.Diagnostics.CodeAnalysis;

namespace Models
{
    public partial class Drawing : DrawingBase
    {
        [SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
        public Drawing()
        {
            DrawingLayer = new HashSet<DrawingLayer>();
            //DrawingVTemplate = new HashSet<DrawingVTemplate>();
            ImageGrid = new HashSet<ImageGrid>();
            //PaperSize = new HashSet<PaperSize>();
            TaskDuration = new HashSet<TaskDuration>();
        }

        //public virtual Project Project { get; set; }
        public virtual ICollection<DrawingLayer> DrawingLayer { get; set; }
        //public virtual ICollection<DrawingVTemplate> DrawingVTemplate { get; set; }
        public virtual ICollection<ImageGrid> ImageGrid { get; set; }
        //public virtual ICollection<PaperSize> PaperSize { get; set; }
        public virtual ICollection<TaskDuration> TaskDuration { get; set; }

        public ImageUpload ToImageUpload()
        {
            ImageUpload imgU = new ImageUpload
            {
                AmazonConfidence = AmazonConfidence,
                BlobURL = BlobURL,
                ContainerName = ContainerName,
                CreatedDate = CreatedDate,
                DrawingId = DrawingId,
                File = File,
                FileName = FileName,
                GoogleConfidence = GoogleConfidence,
                AzureConfidence = AzureConfidence,
                Processed = Processed,
                ProfileVersionId = ProfileVersionId,
                ProjectId = ProjectId,
                Success = Success,
                Type = Type,
                Tasks = Tasks,
                UserId = UserId
            };
            return imgU;
        }
    }
}

File: DataSummitSolution/DataSummitUI/src/app/properties/models/drawing.ts
export class Drawing {

    DrawingId: number
    FileName: string
    BlobUrl: string
    ContainerName: string
    Success: boolean
    ProjectId: number
    ProfileVersionId?: number
    AzureConfidence?: number
    GoogleConfidence?: number
    AmazonConfidence?: number
    Processed?: boolean
    CreatedDate?: Date
    UserId?: number
    Type: string
    UploadDuration?: number
    File?: FormData
    PaperSizeId: number
    PaperOrientationId: number

    constructor(drawingId?: number, name?: string, fileName?: string, 
                blobName?: string, success?: boolean, projectId?: number, 
                profileVersionId?: number,  azureConfidence?: number, 
                googleConfidence?: number, amazonConfidence?: number, 
                processed?: boolean, createdDate?: Date, userId?: number, 
                type?: string, uploadDuration?: number, file?: FormData,
                paperSizeId?: number, paperOrientationId?: number)
    {
        this.DrawingId = drawingId;
        this.ContainerName = name;
        this.FileName = fileName;
        this.BlobUrl = blobName;
        this.Success = success;
        this.ProjectId = projectId;
        this.ProfileVersionId = profileVersionId;
        this.AzureConfidence = azureConfidence;
        this.GoogleConfidence = googleConfidence;
        this.AmazonConfidence = amazonConfidence;
        this.Processed = processed;
        this.CreatedDate = createdDate;
        this.UserId = userId;
        this.Type = type;
        this.UploadDuration = uploadDuration;
        this.File = file;
        this.PaperSizeId = paperSizeId;
        this.PaperOrientationId = paperOrientationId;
    }
}